pkg/didcomm/transport/media_type.go

```diff
@@ -6,9 +6,10 @@ SPDX-License-Identifier: Apache-2.0
 
 package transport
 
-import "fmt"
-
 const (
+	// MediaTypeRFC0019EncryptedEnvelope is the original media type for DIDComm V1 encrypted envelopes as per
+	// Aries RFC 0019.
+	MediaTypeRFC0019EncryptedEnvelope = "JWM/1.0"
```

Comment:
I believe we also need to add media [profiles](https://github.com/hyperledger/aries-rfcs/tree/master/features/0044-didcomm-file-and-mime-types#defined-profiles) as well

Reply:
I've added them in the `outofband` package.

Reply:
I noticed @llorllale updated the pr specifically for media profiles. It looks good now.